src/components/shared/modals/ClaimUnclaimedTokensModal.tsx

```diff
@@ -55,13 +65,13 @@ export default function ConfirmUnstakeTokensModal({
     : false
 
   return (
-    <Modal
+    <TransactionModal
```

Comment:
**praise** for using `TransactionModal`! My fault but we need to specify `transactionPending` prop for it to do anything. Do you want to add that, and also update the type of `transactionPending` prop to `boolean | undefined` to force us to specify it? 🙂

Reply:
dun